src/plusplus.coffee

```diff
@@ -127,7 +127,7 @@ module.exports = (robot) ->
                   "Erased points for #{name}"
       msg.send message
 
-  robot.respond /score (for\s)?(.*)/i, (msg) ->
+  robot.respond /(?:score|karma) (for\s)?(.*)/i, (msg) ->
```

Comment:
This may work better as a config var, which would give you even more flexibility.